### DISCUSSION TITLE
Potplayer

### DISCUSSION THREAD

**Discussion by rollingmoai** [catppuccin/catppuccin#1832] [Port Requests]
**Are you available to help in the creation of this port? 😄 (optional)**
No

**Additional context 📷 (optional)**
@niivu's newly released [Catppuccin theme for Windows](https://www.deviantart.com/niivu/art/Catppuccin-for-Windows-908988670) contains a Potplayer skin that we can use.

**Comment by NoPlagiarism** [comment]
Are you sure this ***Windows*** theme contains PotPlayer skin?
